homeassistant/components/camera/__init__.py

```diff
@@ -375,6 +355,17 @@ def frame_interval(self):
         """Return the interval between frames of the mjpeg stream."""
         return 0.5
 
+    async def create_stream(self) -> Stream:
+        """Create a Stream for stream_source."""
+        # There is at most one stream (a decode worker) per camera
+        if not self.stream:
+            async with async_timeout.timeout(10):
```

Comment:
Realize this timeout value was hard coded before, but maybe we should make this a constant

Reply:
I'll send a follow up to add this.